packages/terra-clinical-detail-view/src/terra-dev-site/doc/clinical-detail-view/ClinicalDetailView.1.doc.mdx

```diff
@@ -0,0 +1,51 @@
+import { Badge } from 'terra-clinical-detail-view/package.json?dev-site-package';
+
+import DetailViewPropsTable from 'terra-clinical-detail-view/src/DetailView?dev-site-props-table';
+import DetailListPropsTable from 'terra-clinical-detail-view/src/DetailList?dev-site-props-table';
+import DetailListItemPropsTable from 'terra-clinical-detail-view/src/DetailListItem?dev-site-props-table';
+
+import DetailViewDivided from '../example/DetailViewDivided?dev-site-example';
+import DetailViewNoDivider from '../example/DetailViewNoDivider?dev-site-example';
+import DetailViewDividedSmallerTitles from '../example/DetailViewDividedSmallerTitles?dev-site-example';
+
+<Badge />
+
+# Terra Clinical DetailView
+
+Detail Views allows you to create a view with text at different levels of importance. A detail view can contain a title, subtitles, a graph, a footer and a list of elements that can display information at a more detailed level.
+
+A Detail List is a structural subcomponent of Detail View indented to be used within the details prop. A Detail List accepts and arranges DetailListItem elements in a flexbox layout that wraps the list of items by row.
+
+Detail List Items are the elements to be placed within a Detail List. A Detail List Item accepts one element item and applies a minimum width of 50% so that the element spans at minimum half of the Detail List. This allows the Detail List to have either one or two items per row depending on the width of the item content in each item.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-clinical-detail-view`
+
+## Usage 
+
+```jsx
+import DetailView from 'terra-clinical-detail-view';
+```
+
+## Component Features
+* [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+* [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
+* [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
+
+[View Component Source Code](https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-detail-view)
+
+## Example 
+<DetailViewDivided title= 'Divided Detail View' />
+<DetailViewNoDivider title= 'Non-Divided Detail View' />
+<DetailViewDividedSmallerTitles title= 'Divided Detail View with Smaller Titles' />
```

Comment:
```suggestion
<DetailViewDivided title="Divided Detail View" />
<DetailViewNoDivider title="Non-Divided Detail View" />
<DetailViewDividedSmallerTitles title="Divided Detail View with Smaller Titles" />
```
I know these arn't linted but we should try to maintain our src code syntax in our examples as well